### PR TITLE
fix(aapd-940): fix navigation issues

### PR DESCRIPTION
**Pull request by Maby0**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/dynamic-forms/questions.js [hunk 1]
```diff
@@ -522,10 +522,10 @@ exports.questions = {
 		html: 'resources/emerging-plan-upload/content.html',
 		documentType: documentTypes.emergingPlanUpload
 	}),
-	uploadOtherRelevantPolicies: new MultiFileUploadQuestion({
+	uploadDevelopmentPlanPolicies: new MultiFileUploadQuestion({
```
Comment (Maby0) [inline packages/forms-web-app/src/dynamic-forms/questions.js:525]: renamed this one as the [copy deck](https://pins-ds.atlassian.net/wiki/spaces/AAPDS/pages/1299578884/DELETED+CD44+-+Upload+any+other+relevant+policies+v14) for the old one was marked as deleted.

File: packages/forms-web-app/src/dynamic-forms/s78-questionnaire/journey.js [hunk 2]
```diff
@@ -114,14 +114,13 @@ class S78Journey extends Journey {
 				.addQuestion(questions.representationUpload)
 				.withCondition(questionHasAnswer(questions.representationsFromOthers, 'yes')),
 			new Section('Planning officer’s report and supporting documents', 'planning-officer-report')
+				.addQuestion(questions.uploadDevelopmentPlanPolicies)
```
Comment (Maby0) [inline packages/forms-web-app/src/dynamic-forms/s78-questionnaire/journey.js:117]: reordered this as per the request in this [navigation correction document](https://pins-ds.atlassian.net/jira/software/c/projects/AAPD/boards/213?selectedIssue=AAPD-940)

File: packages/forms-web-app/src/dynamic-forms/questions.js (hunk 1 above)
Comment (bryn-pins) [inline packages/forms-web-app/src/dynamic-forms/questions.js:525]: if we change this, we should probably update the document type

Comment (bryn-pins) [reply packages/forms-web-app/src/dynamic-forms/questions.js:525]: confusing not correct in: https://pins-ds.atlassian.net/browse/AAPD-783

**Review by bryn-pins — APPROVED** [review]

**Pull request merged**